src/scripts/libraryBrowser.js

```diff
@@ -119,7 +119,10 @@ export function getQueryPagingHtml (options) {
 }
 
 export function showSortMenu (options) {
-    require(['dialogHelper', 'emby-radio'], function (dialogHelper) {
+    Promise.all([
+        import('dialogHelper'),
+        import('emby-radio')
+    ]).then(({default: dialogHelper}) => {
```

Comment:
`Promise.all` result as an array
```suggestion
    ]).then(([{default: dialogHelper}]) => {
```